packages/tests/integration/src/setupTestEnv.ts

```diff
@@ -2,5 +2,7 @@ import { GetEnvVars } from "env-cmd";
 
 export default async function globalSetup() {
   process.env.TZ = "UTC";
+  // process.env.DEBUG = "knex:query,bindings";
+  // process.env.DEBUG = "knex:*";
```

Comment:
question: Was this meant to be left in?

Reply:
Yeah, I hadn't used it before, and wanted to leave it in as a reminder to myself + quick "just uncomment this".

Eventually thinking Joist could support the same sort of thing, i.e. what is now some of our `setFieldLogging(true)` type flags could instead (or also) be like `DEBUG=joist:fields` type settings. :thinking: 

I think the `DEBUG=...` approach is fairly idiomatic within the node ecosystem, or at least was at some point, it seems as little more OG, but not sure what the new alternative is atm.